temporal/api/cloud/cloudservice/v1/request_response.proto

```diff
@@ -212,6 +212,53 @@ message DeleteNamespaceResponse {
     temporal.api.cloud.operation.v1.AsyncOperation async_operation = 1;
 }
 
+message FailoverNamespaceRequest {
+    // The namespace to failover.
+    string namespace = 1;
+    // The target region to failover.
+    temporal.api.cloud.region.v1.Region target_region = 2;
```

Comment:
I would just use a region_id like we do on the namespace spec.

Reply:
Should we migrate the namespace spec to Region? rather than move this to a string region id?